src/ribbon-fill.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
+  <defs>
```

Comment:
• remove `<defs> ... </defs>` wrapper and just keep the `path`
• remove `<use ... />` tag below
• remove `id` attribute on `path`

Reply:
Cleaned it up